gi_wrapper_cpp/dicom_tag_strategies.cpp

```diff
@@ -66,69 +74,50 @@ bool omitSpecifiedTagsStrategy(int tagGroupId, int tagTagId, set <pair<int, int>
 // 1 or other integer values - rewrite all tags from source dicom
 tagsRewriteStrategy resolveTagsRewriteStrategy(int strategy)
 {
-    switch(strategy)
+    switch (strategy)
     {
-        case 5:
-            return omitSpecifiedTagsStrategy;
-        case 4:
-            return rewriteSpecifiedTagsStrategy;
-        case 3:
-            return rewriteAllNonPrivateTagsStrategy;
-        case 2:
-            return rewriteNoTagsStrategy;
-        default:
-            return rewriteAllTagsStrategy;
+    case 5:
+        return omitSpecifiedTagsStrategy;
+    case 4:
+        return rewriteSpecifiedTagsStrategy;
+    case 3:
+        return rewriteAllNonPrivateTagsStrategy;
+    case 2:
+        return rewriteNoTagsStrategy;
+    default:
+        return rewriteAllTagsStrategy;
     }
 }
 
-void applyDicomTagsRewriteStrategy(DataSet* source, DataSet* destination, int rewriteTagsStrategyId, set <pair<int, int>> userTags)
+void applyDicomTagsRewriteStrategy(DataSet& source, DataSet& destination, int rewriteTagsStrategyId, set <pair<int, int>> userTags)
```

Comment:
@kdbogusz please pass userTags by const reference - current signature causes unnecessary copy of potentially big data structure (const set<pair<int, int>>& userTags)

Reply:
This should apply to all places where such data structures are passed (including unchanged code)